### ISSUE TITLE
OrthographicTrackballControls handleResize() does not work

### ISSUE THREAD

**Issue by klebba** [mrdoob/three.js#5842]
The constructor configures these properties which are essential for zooming:

```
this.left0 = this.object.left;
this.right0 = this.object.right;
this.top0 = this.object.top;
this.bottom0 = this.object.bottom;
```

`handleResize` should update these properties, but doesn't. Configuring them externally/manually after updating my OrthographicCamera bounds fixes the issue:

```
// in my resize handler, controls is an instance of THREE.OrthographicTrackballControls
controls.left0 = this.controls.object.left;
controls.right0 = this.controls.object.right;
controls.top0 = this.controls.object.top;
controls.bottom0 = this.controls.object.bottom;
```

Using the latest here: https://github.com/mrdoob/three.js/blob/9e0012b9693aa8b0247abe6734f5693f59491ae0/examples/js/controls/OrthographicTrackballControls.js

I think this is an easy fix, please take a look @patrickfuller 

Thanks!

**Comment by patrickfuller** [comment]
@klebba I can't reproduce your use case, so the pull request is currently untested. Would you mind trying out the `OrthographicTrackballControls` on my fork and seeing if it solves your problem?

**Comment by klebba** [comment]
@patrickfuller -- yes, the version on the `ortho` branch of your fork works. Here's a demo showcasing the bug with the current version in `master`:

http://jsfiddle.net/klebba/h27avkpa/

If you change the external dependency to your version, it works fine.

Thanks!

**Comment by patrickfuller** [comment]
Had to increase the clipping planes on the camera, but I reproduced the error. Switching to my fork fixed the problem.

@mrdoob I'm okay with this pull request.

**Comment by dubejf** [comment]
Fixed by #5854
